### PR TITLE
docs: update messaging around React first

### PR DESCRIPTION
**Pull request by alisonjoseph**
Updated the paragraph on the "other frameworks" page along with the page descriptions. 

Do we want to add this paragraph to a more prominent place on the website also? 

> The Carbon Design System is built React first. We also support core parts of the system in vanilla JS, Angular, and Vue.

### CODE REVIEW

**Comment by janchild** [comment]
@alisonjoseph, I think we could also add this info as a third paragraph on this page. https://www.carbondesignsystem.com/get-started/about-carbon

We could add the sentence about other frameworks with a link back to the other page. Sound good to you?

**Comment by alisonjoseph** [comment]
makes sense @janchild, just updated that page.

File: src/pages/get-started/develop/other-frameworks.mdx [hunk 1]
```diff
@@ -26,7 +26,7 @@ The Carbon Design System supports vanilla JS, React and Angular as core parts of
 
 ### Using just the styles
 
-Developers wanting to build in different ways follow the instructions for the [Vanilla](/get-started/develop/vanilla) library to access the styles and build out their own components.
+Developers wanting to build in different ways follow the instructions for the [vanilla JS](/get-started/develop/vanilla) library to access the styles and build out their own components.
```
Comment (janchild) [inline src/pages/get-started/develop/other-frameworks.mdx]:
```suggestion
Developers wanting to build in different ways, follow the instructions for the [vanilla JS](/get-started/develop/vanilla) library to access the styles and build out their own components.
```

File: src/pages/get-started/about-carbon.mdx [hunk 2]
```diff
@@ -24,6 +24,8 @@ As IBM's official design system, Carbon serves a wide range of designers and dev
 
 The goals of the design system include improving UI consistency and quality, making the design and development process more efficient and focused, establishing a shared vocabulary between designer and developer, and providing clear, discoverable guidance around design and development best practices.
 
+The design system is built [React](/get-started/develop/react) first. We also support core parts of the system in [vanilla JS](/get-started/develop/vanilla), [Angular](/get-started/develop/angular), and [Vue](/get-started/develop/vue). If you’re you’re using a different framework, you can still build components by following our guidelines for [other frameworks](/get-started/develop/other-frameworks).
```
Comment (janchild) [inline src/pages/get-started/about-carbon.mdx]:
```suggestion
The design system is built [React](/get-started/develop/react) first. We also support core parts of the system in [vanilla JS](/get-started/develop/vanilla), [Angular](/get-started/develop/angular), and [Vue](/get-started/develop/vue). If you’re using a different framework, you can still build components by following our guidelines for [other frameworks](/get-started/develop/other-frameworks).
```

**Pull request merged**